tools/lint

```diff
@@ -595,7 +595,10 @@ def build_custom_checkers(by_lang):
             "docs/migration-renumbering.md",
             "docs/readme-symlink.md",
             "README.md",
+            "zerver/webhooks/helloworld/doc.md",
+            "zerver/webhooks/solano/doc.md",
             "zerver/webhooks/trello/doc.md",
+            "zerver/webhooks/papertrail/doc.md",
```

Comment:
Oh, and also, I think it'd be reasonable to line-wrap some of the curl commands.